guides/micronaut-jms-aws-sqs/micronaut-jms-aws-sqs.adoc

```diff
@@ -0,0 +1,121 @@
+common:header.adoc[]
+
+common:requirements.adoc[]
+* An https://aws.amazon.com/[AWS account] with:
+** An IAM user with enough permissions to create and manage a queue instances in SQS.
+** The AWS CLI configured to use the IAM user above.
+
+common:completesolution.adoc[]
+
+common:create-app-features.adoc[]
+
+== Micronaut JMS SQS Dependency
+
+To use the Micronaut JMS SQS integration, add the following dependency:
```

Comment:
The dependency is added by the `jms-sqs` feature in the create-app command. Either remove this section, or change it to point out to the user that the feature adds this dependency. That's probably not needed though - they can see the effect of using the feature by clicking the diff link in the "If you have an existing Micronaut application ..." section

Reply:
Good point